src/utils/applyPlugins.js

```diff
@@ -1,2 +1,2 @@
-const applyPlugins = (data, plugins = []) => plugins.reduce((d, plugin) => plugin(d), data);
+const applyPlugins = (data, plugins = [], requestUrl) => plugins.reduce((d, plugin) => plugin(d, requestUrl), data);
```

Comment:
We could also pass the domain (and action) to the plugin instead (and also move the creation of the url down to the `request` file) that way we can avoid writing that function(`getRequestDomain`) in order to extract the domain name from the url. Because we just created the url using the domain and action. What do you think?

Reply:
I think that could work. It would prevent us from getting a piece of information out of something we just constructed. It also makes sense for me that the url get's created in the request function.